editor/views/rai.py

```diff
@@ -0,0 +1,262 @@
+                value=metadata.data_annotation_protocol,
+                on_change=handle_rai_change,
+                args=(RaiEvent.RAI_DATA_ANNOTATION_PROTOCOL, metadata, key),
+            )
+            key = "metadata-data-annotation-platform"
+            st.text_area(
+                label=(
+                    "**Platform**. Platform, tool, or library used to collect annotations by human annotators"
+                ),
+                key=key,
+                value=metadata.data_annotation_platform,
+                on_change=handle_rai_change,
+                args=(RaiEvent.RAI_DATA_ANNOTATION_PLATFORM, metadata, key),
+            )
+            key = "metadata-data-annotation-analysis"
+            st.text_area(
+                label=(
+                    "**Analysis**. Considerations related to the process of converting the “raw” annotations into the labels that are ultimately packaged in a dataset - Uncertainty or disagreement between annotations on each instance as a signal in the dataset, analysis of systematic disagreements between annotators of different socio demographic group,  how the final dataset annotations will relate to individual annotator responses"
+                ),
+                key=key,
+                value=metadata.data_annotation_analysis,
+                on_change=handle_rai_change,
+                args=(RaiEvent.RAI_DATA_ANNOTATION_ANALYSIS, metadata, key),
+            )
+            key = "metadata-data-annotation-demographics"
+            st.text_area(
+                label=(
+                    "**Demographics**. List of demographics specifications about the annotators"
+                ),
+                key=key,
+                value=metadata.annotator_demographics,
+                on_change=handle_rai_change,
+                args=(RaiEvent.RAI_DATA_ANNOTATION_DEMOGRAPHICS, metadata, key),
+            )
+            key = "metadata-data-annotation-tools"
+            st.text_area(
+                label=(
+                    "**Tools**. List of software used for data annotation ( e.g. concept extraction, NER, and additional characteristics of the tools used for annotation to allow for replication or extension)  "
+                ),
+                key=key,
+                value=metadata.machine_annotation_tools,
+                on_change=handle_rai_change,
+                args=(RaiEvent.RAI_DATA_ANNOTATION_TOOLS, metadata, key),
+            )
+            key = "metadata-data-annotation-per-item"
+            st.text_area(
+                label=(
+                    "**Annotation per item**. Number of human labels per dataset item  "
+                ),
+                key=key,
+                value=metadata.annotation_per_item,
+                on_change=handle_rai_change,
+                args=(RaiEvent.RAI_DATA_ANNOTATION_PER_ITEM, metadata, key),
+            )
+        with st.expander("**Data Preprocessing**", expanded=False):
+            oneToManyProperty(
```

Comment:
In Python, we recommend using snake_case over camelCase

Reply:
Changed.